gemini/use-cases/retrieval-augmented-generation/small_to_big_rag/small_to_big_rag.ipynb

```diff
@@ -0,0 +1,848 @@
+{
```

Comment:
<p>There are more efficient ways to run functions on dataframes. For example, <a href="https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.apply.html" target="_blank">https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.apply.html</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/485/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
Hi Holt, apologies, which line are you pointing to?